### PR TITLE
Issue #15218: Formatted input files of section 6.4 Finalizers to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

**Comment by Zopsss** [comment]
Circle CI failing, maybe restarting CI should fix it

**Comment by romani** [comment]
CI is restarted

File: src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizer.java [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
 
-public class InputNoFinalizer
-{
-    public void finalize() // violation 'Avoid using finalizer method.'
-    {
-        // It's not enough to check if the METHOD_DEF branch contains a PARAMETER_DEF, as that would
-        // treat this method as having a parameter.
-        Runnable runnable = new Runnable() {
+/** Some javadoc. */
+public class InputNoFinalizer {
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizer.java:4]: @Zopsss , please try to not add javadocs, and simply format.
I plan to persuade mentors to allow single review merge on just formatting.

**Review by romani — APPROVED** [review]
ok to merge

File: src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizer.java (hunk 1 above)
Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizer.java:4]: Eventually , yrs, but please do not complicate formatting updates. We should merge all formatting quickly.

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizer.java:4]: got it, will keep this in mind

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**